### DISCUSSION TITLE
爱门圣经

### DISCUSSION THREAD

**Discussion by elysiam2** [share121/inter-knot#59] [其他]
![1](https://img.hongyoubizhi.com/picture/pages/regular/2023/10/25/13/119360040_p0_master1200.jpg?x-oss-process=image/resize,m_fill,w_1000)
<span style="color: #ff8cc6">让无人传颂的歌谣，再度启唱。</span>
<span style="color: #ff8cc6">为历史尘封的故事，续写新章。</span>
<span style="color: #ff8cc6">高洁而纯真的灵魂，踏上旅途。</span>
<span style="color: #ff8cc6">愿掠集之兽爱佑其身，赐予「空梦」的自由。</span>
<span style="color: #ff8cc6">愿渡尘之羽爱佑其身，赐予「浮生」的坚忍。</span>
<span style="color: #ff8cc6">愿绘世之卷爱佑其身，赐予「繁星」的纯真。</span>
<span style="color: #ff8cc6">愿噬界之蛇爱佑其身，赐予「无限」的渴求。</span>
<span style="color: #ff8cc6">愿黎明之哨爱佑其身，赐予「旭光」的意志。</span>
<span style="color: #ff8cc6">愿寸断之刃爱佑其身，赐予「刹那」的决绝。</span>
<span style="color: #ff8cc6">愿善法之瞳爱佑其身，赐予「天慧」的智识。</span>
<span style="color: #ff8cc6">愿坏劫之焱爱佑其身，赐予「鏖灭」的赤诚。</span>
<span style="color: #ff8cc6">愿愚戏之匣爱佑其身，赐予「螺旋」的奇迹。</span>
<span style="color: #ff8cc6">愿璀耀之歌爱佑其身，赐予「黄金」的光辉。</span>
<span style="color: #ff8cc6">愿深罪之槛爱佑其身，赐予「戒律」的慈爱。</span>
<span style="color: #ff8cc6">愿无烬之剑爱佑其身，赐予「救世」的理想。</span>
<span style="color: #ff8cc6">愿无瑕之人爱佑其身，赐予「真我」的回归。</span>
<span style="color: #ff8cc6">我们于此赞颂，始源之律者的诞生                   「我这支歌将抚摸你的前额，犹如那祝福的亲吻」</span>
<span style="color: #ff8cc6">「当你独自一人时，它会坐在身旁与你耳语；当你陷入人群时，它又会保护你远离喧嚣」</span>
<span style="color: #ff8cc6">「我的歌将成为你梦想的羽翼，它将载着你的心到那未知的边缘」</span>
<span style="color: #ff8cc6">「当黑夜遮蔽了你的路时，它又成为了照耀在你头顶的忠实星光」</span>
<span style="color: #ff8cc6">于是，「始源」的故事迎来落幕</span>
<span style="color: #ff8cc6">于是，「始源」的故事迎来伊始</span>
<span style="color: #ff8cc6">「以我为终……」</span>
<span style="color: #ff8cc6">以我……为始。</span>
<span style="color: #ff8cc6">爱莉神教无需对我教多言，获神之爱者皆是我教教徒，但却须知神爱世人</span>
<span style="color: #ff8cc6">即世人皆是我教教徒</span>
<span style="color: #ff8cc6">教旨其一：满足神的一切愿望，不计代价</span>
<span style="color: #ff8cc6">教旨其二：得神之爱，世人就应当为其满足</span>
<span style="color: #ff8cc6">教旨其三：须知神乃圣洁且无暇，教徒也应保持此等高洁，切勿让神因为教徒缘由而沾上污秽</span>
<span style="color: #ff8cc6">教旨其四：请保持健康阳光，积极向上的生活态度，因为神乐见世人如此</span>
<span style="color: #ff8cc6">教旨其五：神虽无暇却易感寂寞，教徒应在每晚睡前默念神之圣名五遍</span>
<span style="color: #ff8cc6">教旨其六：见花见鸟如见神明，因神爱戴众生，请尽量亲切呵护身边的各类小生命。</span>
<span style="color: #ff8cc6">教旨其七：神爱世人，且钟爱长角之人，还没有角的教徒请尽力长出角来，勿让教会蒙羞</span>
<span style="color: #ff8cc6">教旨其八：虽神爱世人，却有心怀叵测之人对神不敬，妄自对神加以胡思乱想，此为大不敬。但教徒亦须知，神爱世人，包括这妄想连篇之人，所以神会原谅他，如同原谅你我之罪。</span>
<span style="color: #ff8cc6">教旨其九：不可与人争斗，神哀于人们无休止的斗争漩涡，切记神会原谅任何人，教徒应以神之旨意渡他人</span>
<span style="color: #ff8cc6">愿神之爱长存你我，爱门~ </span>

**Comment by MouxiuSama** [comment]
图片没加载出来哦~

**Reply by elysiam2** [reply]
现在好了

**Comment by Loxic123** [comment]
爱门！

**Comment by myt521** [comment]
爱门

**Comment by dsttsz** [comment]
爱门！

**Comment by jidianzhuxing** [comment]
爱门

**Comment by wryyy123** [comment]
爱门！

**Comment by 54410** [comment]
爱门

**Comment by Aixi-er** [comment]
爱门

**Comment by yemu214** [comment]
爱门

**Comment by changline** [comment]
爱门！

**Comment by wxxwx** [comment]
爱门

**Comment by Elysia-Ego-093** [comment]
爱门！

**Comment by Liangjl-2024** [comment]
爱门永存！！！

**Comment by akatuona** [comment]
爱门！

**Comment by Hexiaoling457** [comment]
爱门！！！

**Comment by LB614-111** [comment]
爱门！

**Comment by buluoniya21** [comment]
爱门

**Comment by bamboo514** [comment]
爱门！

**Comment by 1376881** [comment]
爱门！

**Comment by kokomi0410** [comment]
爱门！

**Comment by HAZUUU-ctrl** [comment]
爱莉希雅是最好的！！！！

**Comment by long7746** [comment]
爱门

**Comment by jjujij** [comment]
爱门！

**Comment by YeMengXue** [comment]
爱门！

**Comment by xyc2718** [comment]
爱门!

**Comment by School574** [comment]
爱门！！！

**Comment by Teriri-Apocalypse** [comment]
爱门~

**Comment by RozenSchicksal** [comment]
✝爱门✝

**Comment by dong200497** [comment]
爱门

**Comment by wushang114** [comment]
不赖，写得好

**Comment by luoyu007** [comment]
爱门

**Comment by xyy18030223854** [comment]
爱门

**Comment by shishenyihe** [comment]
爱门

**Comment by wsffdg-123** [comment]
爱门

**Comment by Leepeak0531** [comment]
爱门

**Comment by Zhishirur** [comment]
爱门！

**Comment by liweibin089** [comment]
爱门！！！！！

**Comment by Dark-Master404** [comment]
爱门

**Comment by Dark-Master404** [comment]
爱门

**Comment by JinYi-qq** [comment]
爱门！

**Comment by z1125801** [comment]
爱门

**Comment by kianaS0** [comment]
爱门！

**Comment by baimo2004** [comment]
爱门

**Comment by Achayyyy** [comment]
爱门！

**Comment by SAKURA-DD** [comment]
爱门

**Comment by Phaeton9528** [comment]
爱门

**Comment by lappslover** [comment]
爱门

**Comment by moonhalo7** [comment]
爱门

**Reply by TiffanyHSR** [reply]
爱门

**Comment by IX-11** [comment]
爱门！

**Comment by Guivov** [comment]
这个委托我接了!

**Comment by jdjwzx233** [comment]
爱门！！！！

**Comment by ZJX423** [comment]
你这名字和我前面一样了

**Comment by dududadududa-max** [comment]
爱门！

**Comment by misaka14458** [comment]
爱门！

**Comment by WORLD-executeME** [comment]
爱门！

**Comment by dianziluyu** [comment]
爱门

**Comment by Playmaker-Hai** [comment]
爱门！

**Comment by Xyy12345689** [comment]
爱门

**Comment by ssdxs15** [comment]
爱门
![3E63EA1E](https://github.com/user-attachments/assets/72ec926f-8aee-49e3-975b-04719183e90b)

**Reply by siler-wolf** [reply]
爱门！~

**Comment by huashuideryu** [comment]
爱门！

**Comment by 117518** [comment]
爱门！

**Comment by Changes1794** [comment]
爱门！

**Comment by AL-Pinecore** [comment]
爱门！

**Comment by nahida0-o** [comment]
爱门！

**Comment by Elysia379** [comment]
爱门

**Comment by Elysia379** [comment]
爱门（合十）

**Comment by ghost** [comment]
愛門

**Comment by ccyhyym** [comment]
爱门

**Comment by MiaoJiSir** [comment]
爱门

**Comment by YiMie148** [comment]
爱门

**Reply by Elylq** [reply]
爱门

**Comment by 521888zX** [comment]
爱门！

**Comment by circle-P** [comment]
爱门

**Comment by udigjbx114514** [comment]
爱门

**Comment by Railgun11313** [comment]
爱门

**Comment by ShuoChen08** [comment]
爱门

**Comment by light814** [comment]
爱门!

**Comment by Uceck** [comment]
爱门

**Comment by ConsistDust3838** [comment]
爱门

**Comment by ANDWULI** [comment]
爱门！

**Comment by Elysia-Ego-093** [comment]
![Image_1722092297722](https://github.com/user-attachments/assets/5858c060-db0d-42dd-a0d8-2ab96aeb164e)
我永远喜欢爱莉希雅！

**Reply by aomengxin** [reply]
好图，拿了感谢［崩坏三 光辉矢愿 比心］

**Comment by ZhanPai123** [comment]
爱门

**Comment by mixue4646** [comment]
爱门伟大，无需多言

**Comment by 1122544** [comment]
爱门

**Comment by LuinEmiya** [comment]
爱门！

**Comment by BOXANK** [comment]
爱门永存！！！！！！！

**Comment by Fairy561** [comment]
爱门

**Comment by 75020-maker** [comment]
爱门！！！！！！！！！！

**Comment by Belle114514** [comment]
爱门！

**Comment by Bilistar333** [comment]
爱门！！！

**Comment by FuningnaFurina** [comment]
爱门！

**Comment by 15976724736** [comment]
爱门！

**Comment by Eren-Ygler** [comment]
爱门！

**Comment by kailunerr** [comment]
爱门！

**Comment by hjchjchjchjc707** [comment]
爱门！

**Comment by ReALiv** [comment]
爱门伟大，无需多言

**Comment by Lunaum-chen** [comment]
爱门

**Comment by cheese502** [comment]
爱门！

**Comment by lizard315315** [comment]
爱门！

**Comment by yuleiiiiii** [comment]
🙏爱门！🙏

**Comment by Lanxey** [comment]
爱门🙏

**Comment by Griffin233** [comment]
标题的语法是不是错了，style不是应该单独写吗，应该是<span color="#FF1234">之类的（html新手单纯问问）

**Comment by Elysia-lover** [comment]
爱门

**Comment by aaa00073** [comment]
我也好想跟爱莉希雅一起姓爱啊！

**Reply by Mayf1ower** [reply]
你逆天

**Comment by Bochiii984** [comment]
爱门！

**Comment by infinite-vector** [comment]
爱门！🙏🙏

**Comment by th-moon** [comment]
爱门！

**Comment by Mayf1ower** [comment]
世人都称颂我爱伟大！啊，爱门！

**Comment by JiLingZi** [comment]
爱门！！！~

**Comment by mroving** [comment]
爱门！

**Comment by arkdecade** [comment]
爱门

**Comment by nailbiting21** [comment]
爱门

**Comment by xinqintian** [comment]
爱门

**Comment by jiishdhhwhahysyd** [comment]
爱门！

**Comment by kimi12123** [comment]
爱门！

**Comment by w1-w2-w3** [comment]
爱门！

**Comment by Larksonguesinaspic** [comment]
#include<bits/stdc++.h>
using namespace std;
int main(){
     printf("爱门")；
     return 0；
}
![image](https://github.com/user-attachments/assets/e6a79436-e95c-48f1-80f9-3dc522aab8de)

**Comment by 765hi** [comment]
爱门！

**Comment by BillyYesSir** [comment]
爱门！

**Comment by yunnikuu** [comment]
爱门！

**Comment by AKT505** [comment]
设置字体颜色的代码被ban了？

**Comment by kevin-114514** [comment]
爱门!

**Comment by Qiling-XwX** [comment]
爱门！
![OIP-C](https://github.com/user-attachments/assets/a6c38f83-b88b-4f2a-b5d0-8c7a700f3f8b)

**Comment by sinke122** [comment]
爱门

**Comment by yifan-cpu** [comment]
爱门

**Comment by shuyishaoxiancao** [comment]
我永远喜欢爱莉希雅！

**Comment by SonwNight3089** [comment]
爱门！

**Comment by kimile30** [comment]
爱门

**Comment by SWYH34827** [comment]
爱门！！！

**Comment by KamisatoAyaka-Alt** [comment]
![17226169317619c1afae0ec36dda9353ac576a1f094be83ec4abea2500fa5781001d4994962d5 0](https://github.com/user-attachments/assets/9179734e-5f1f-4a38-9a58-856adf544e57)
爱门

**Comment by GanlinSama** [comment]
爱门！

**Comment by share121** [comment]
爱门！

**Comment by Fairy561** [comment]
爱门！

**Comment by 13579yangh** [comment]
爱门！

**Comment by time-p** [comment]
爱门

**Comment by RayWould** [comment]
爱门！

**Comment by WeiAi2333** [comment]
爱门！

**Comment by WorldlyTree** [comment]
爱门！

**Comment by sjk428** [comment]
爱门！

**Comment by Jiucheng258** [comment]
我敲，叠了100多楼，这就是爱门吗。。。
爱门！

**Comment by qepe** [comment]
爱门!

**Comment by yugpooo** [comment]
爱门！

**Comment by aiTunes-dev** [comment]
看起来标题不支持HTML（

**Comment by Loxia1121** [comment]
♪爱门♪

**Comment by Buer-Nahida** [comment]
@HeroBrine-c 帮你Subscribe了，不用谢

**Comment by HVERNH** [comment]
✝爱门✝

**Comment by GUTSTPC** [comment]
爱门！

**Comment by Ste11er** [comment]
爱门!

**Comment by 114514zhou** [comment]
我永远喜欢爱莉希雅！！！爱门！！

**Comment by Kwazii1** [comment]
爱门

**Comment by xtyS126** [comment]
爱门！

**Comment by 147258huan** [comment]
爱门♬

**Comment by Oaqn** [comment]
爱门！

**Comment by Aixi-er** [comment]
爱门

**Comment by adsadk12** [comment]
爱门！

**Comment by shinbako** [comment]
爱门！！

**Comment by zwz2732629957** [comment]
爱门！！

**Comment by lamtt26** [comment]
爱门

**Comment by paowaer566** [comment]
爱门

**Comment by 12344856** [comment]
爱门！

**Comment by breeze0317** [comment]
爱门！

**Comment by Fuklos** [comment]
爱门！

**Comment by WorldZHT** [comment]
爱门！

**Comment by li743** [comment]
爱门！！

**Comment by Landseasky** [comment]
爱门！

**Comment by yue0ying** [comment]
爱门！！！！

**Comment by Callkit-P0p** [comment]
爱门！

**Comment by lllliufeng** [comment]
爱门

**Comment by sigsnsishmsishmwibdm** [comment]
爱爱爱爱门！

**Comment by woqiyounai1** [comment]
爱门

**Comment by SWYH34827** [comment]
爱门

**Comment by yileinafs** [comment]
爱门

**Comment by Lingyao-cangxuan** [comment]
爱门

**Comment by TOKLK** [comment]
绳网真是太棒啦，这里居然也有爱门的人

**Comment by ycl73** [comment]
爱门

**Comment by whatcanisaylaoda** [comment]
标题有乱码。

**Comment by ahjshysu188** [comment]
爱门

**Comment by Qiling-XwX** [comment]
爱门！

**Comment by Elysia16** [comment]
爱门

**Comment by ELYYLE** [comment]
爱门

**Comment by RuomoPao** [comment]
爱门🙏

**Comment by shenwang52321** [comment]
爱门永存！！！！

**Comment by Rainy-poi** [comment]
爱门！！！

**Comment by xiaowang66788** [comment]
爱门！！！

**Comment by LANKgilu** [comment]
爱门！

**Comment by qize6665** [comment]
爱门永存!!!

**Comment by aimen114514** [comment]
爱门

**Comment by SeeleVollereid** [comment]
爱门【补档】(因为大号不小心被我玩没了@Seele1st)

**Comment by Phaethon-0019** [comment]
爱门！！！

**Comment by A125c** [comment]
爱门

**Comment by xiya10086** [comment]
爱门!<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/254352b8702fbdfe4b51919568b16f23_2308445851003216507.gif" alt="爱莉希雅-吃雪糕" width="75" >

**Comment by wshqdd** [comment]
爱门！

**Comment by ljydq123** [comment]
爱门

**Comment by Librarykk** [comment]
爱门！！<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/254352b8702fbdfe4b51919568b16f23_2308445851003216507.gif" alt="爱莉希雅-吃雪糕" width="75" >

**Comment by Pinuttt** [comment]
爱门！<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/94252e623d6ce6c3a2463be2afc23751_1000866240408823916.gif" alt="undefined" width="75" >

**Comment by shenliwlp** [comment]
爱门！！！

**Comment by 6078HITHEDAY** [comment]
你这ｍａｒｋｄｏｗｎ语法，_github的ｍａｒｋｄｏｗｎ语法规范有一点点不同_ 要学习一下

**Comment by sj49193646** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >爱门

**Comment by xjf0326** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/254352b8702fbdfe4b51919568b16f23_2308445851003216507.gif" alt="爱莉希雅-吃雪糕" width="75" >爱门！

**Comment by kbchino** [comment]
爱门

**Comment by lyc736068585** [comment]
爱门

**Comment by MrDucker** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >爱门

**Comment by silver-fox217** [comment]
爱门永存！

**Comment by hk0852** [comment]
爱门【爱心】

**Comment by TairirispuerQ** [comment]
爱门！！！

**Comment by bailanren13** [comment]
爱门<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/94252e623d6ce6c3a2463be2afc23751_1000866240408823916.gif" alt="爱莉希雅-扭扭" width="75" >

**Comment by haily12345** [comment]
爱门

**Comment by mingyu818** [comment]
爱门

**Comment by hua-yuanshuai** [comment]
爱门<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/eb427fe8d802a27e82e694e7ee400bc3_7459590478493659790.gif" alt="符华-变身" width="75" >

**Comment by ysd1911** [comment]
爱门！ ！！！

**Comment by Brighttaz** [comment]
爱门！！！！

**Comment by DUODUOyyds6** [comment]
爱门！

**Comment by IHushuan** [comment]
爱门！<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Comment by hyperion-netizen** [comment]
爱门！

**Comment by DaqiaoJO** [comment]
爱门！

**Comment by Occult-Hacker** [comment]
爱门！

**Comment by ytf526** [comment]
太好了是老婆我爽死了！！！！！！

**Comment by Bailinghai** [comment]
爱门！

**Comment by AiMen-3** [comment]
爱门！

**Comment by ghost** [comment]
爱门！
![ailixiya](https://github.com/user-attachments/assets/71f01fcc-f50b-486d-b9d7-103ffc0755c6)

**Comment by Elysia52077** [comment]
爱门

**Comment by dnmdngcsb** [comment]
爱门!
![Uploading 118152409_p0 (1).png…]()

**Comment by Maple013** [comment]
爱门<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/94252e623d6ce6c3a2463be2afc23751_1000866240408823916.gif" alt="爱莉希雅-扭扭" width="75" >

**Comment by phigros53** [comment]
爱门

**Comment by blacktree-eme** [comment]
爱门

**Comment by diesieqte** [comment]
爱门

**Comment by wrxwyj** [comment]
爱门

**Comment by flamingo0106** [comment]
爱门

**Comment by CJHXAMSTLP** [comment]
爱门

**Comment by Kirolei** [comment]
爱门！

**Comment by yssxrc** [comment]
爱门！

**Comment by siajanajjaan** [comment]
爱门

**Comment by feathink0818** [comment]
爱门<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Comment by amaneyunomi** [comment]
爱门！

**Comment by DrLiSi** [comment]
爱门！

**Comment by paul-way52** [comment]
绝了<img src="https://bbs-static.miyoushe.com/static/2023/10/10/5bc71020b76055d271f9e744daaf880e_5037006894041683415.png" alt="拉尼_晕" width="75" >

**Comment by Momsaysnamesthataretoolongmakefoolslook** [comment]
第一日：赐予粉毛......

**Comment by ym1030917736** [comment]
爱门！

**Comment by Letyem** [comment]
爱门
<img src="https://img-static.mihoyo.com/communityweb/upload/40daa321d201f9f863578e1571fb9b19.png" alt="爱莉希雅-疑惑" width="75" >

**Comment by shengwang999** [comment]
爱门！！！

**Comment by wangyu4444449** [comment]
爱门

**Comment by Kiiro-Elysia** [comment]
爱门

**Comment by UYaXB** [comment]
爱门

**Comment by Samu1KU** [comment]
爱门

**Comment by dfddtabc** [comment]
爱门！！！

**Comment by gxianqshwo** [comment]
爱门！！！

**Comment by moranze** [comment]
爱门

**Comment by raven-null** [comment]
爱门

**Comment by cmcwfxs** [comment]
爱门

**Comment by 6666780** [comment]
爱门！

**Comment by LCactend** [comment]
🙏爱门🙏

**Comment by SeeieKianna** [comment]
爱门!

**Comment by MLGaaa** [comment]
爱门❗

**Comment by mrfhg** [comment]
爱门!

**Comment by udhhe** [comment]
爱门

**Comment by MLG-AT** [comment]
爱门

**Comment by hsdgsdfger** [comment]
爱门！

**Comment by ALXY-12** [comment]
爱门<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/254352b8702fbdfe4b51919568b16f23_2308445851003216507.gif" alt="爱莉希雅-吃雪糕" width="75" >

**Comment by liao-xi** [comment]
爱门

**Comment by azb-666** [comment]
爱门

**Comment by waerte-yang** [comment]
爱门！

**Comment by Orange-Icepop** [comment]
爱门！
话说你这个HTML标签没生效啊（

**Comment by wsz5o88** [comment]
爱门！

**Comment by Vellichor35** [comment]
爱门！<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/94252e623d6ce6c3a2463be2afc23751_1000866240408823916.gif" alt="爱莉希雅-扭扭" width="75" >

**Comment by 9baka99** [comment]
爱门

**Comment by shiyunzhuo2007** [comment]
爱门

**Comment by fzdqz** [comment]
爱门🙏

**Comment by TNCNET** [comment]
爱门🙏

**Comment by 1465505366** [comment]
愛夢

**Comment by Fluorine-z** [comment]
爱门

**Comment by iuhjk** [comment]
爱门<img src="https://upload-bbs.miyoushe.com/upload/2023/01/21/0aea42c72ce1e76e343591bec4afbaaf_8938786851449774348.png" alt="爱莉希雅_来啦" width="75" >

**Comment by JiuXiat** [comment]
爱门

**Comment by Herrscher-of-Origin** [comment]
爱莉爱莉爱？

**Comment by ysd1911** [comment]
爱门 <img src="https://upload-bbs.miyoushe.com/upload/2023/01/21/0aea42c72ce1e76e343591bec4afbaaf_8938786851449774348.png" alt="爱莉希雅_来啦" width="75" >

**Comment by rqrq201704** [comment]
爱门！

**Comment by Jenckmin** [comment]
爱门！

**Comment by DowXing** [comment]
爱门

**Comment by nabeilishi** [comment]
爱门爱门爱门爱门

**Comment by aomengxin** [comment]
爱莉爱莉爱
![IMG_20240919_235602](https://github.com/user-attachments/assets/13f7f439-1620-4b90-817b-364205fc95b2)

**Comment by Loxic123** [comment]
虽然已经发过一条了，但是，我要再发一条来庆祝我抢到了戒指<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >
![dcaf0a2e1a420ca95b187f98e37564b](https://github.com/user-attachments/assets/fee82bdc-e5e3-4767-a234-4fdeed22e53d)

**Reply by ALXY-12** [reply]
(艹皿艹)<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/282acfb8b0dd68bc60862c29f9444feb_7685178076879267260.gif" alt="千劫-气炸" width="75" >

**Comment by 119009116** [comment]
爱门

**Comment by summer293** [comment]
爱门！

**Comment by anbi-demala** [comment]
爱门！
